README.md

```diff
@@ -6,3 +6,15 @@ Simple and easy to use voting app using the django framework and python3.
 
 # Local Setup
 Follow the [Setup instructions here](docs/setup_instructions.md)
+
+# Integrations
+| Name | Description |
+| --- | --- |
+| [Better Code Hub](https://bettercodehub.com/results/miniscruff/votingapp) | High level code analyzer for things like code duplication, automated tests and short functions. |
+| [CodeCov](https://codecov.io/gh/miniscruff/votingapp) | Aggregates code coverage data. |
+| [GitHub](https://github.com/miniscruff/votingapp) | Git source control management |
+| [Heroku](https://dashboard.heroku.com/pipelines/35929301-0ed9-46a7-b52a-e91c8e59d8db) _(Requires Auth)_ | Server hosting platform as a service (PaaS) includes SSL. |
+| [PyUp](https://pyup.io/account/repos/github/miniscruff/votingapp/) | Python dependency tracker and security tool. |
+| [SideCI](https://sideci.com/gh/113536169/news_feed) _(Requries Auth)_ | Static code analyzer for many langauges (using python and css) as well as a spell checker |
```

Comment:
__[Misspell]__

"Requries" is a misspelling of "Requires"


<!-- Comment by [SideCI](https://sideci.com) -->